recipes/glimpse-bio/build.sh

```diff
@@ -1,22 +1,27 @@
 #!/bin/bash
 
+export COMMIT_VERS="${GIT_FULL_HASH}"
```

Comment:
```suggestion
export COMMIT_VERS="${PKG_VERSION}"
```

or we could use `${PKG_VERSION}-${PKG_BUILDNUM}" to have the bioconda build number too.  Thoughts?

See: https://bioconda.github.io/contributor/guidelines.html#link-and-unlink-scripts-pre-and-post-install-hooks

Reply:
I think it should be a good alternative.
We can still afterwards, ask the maintainers of Glimpse to change the way of computing those two variable.